lhotse/dataset/speech_recognition.py

```diff
@@ -18,23 +18,23 @@ class SpeechRecognitionDataset(Dataset):
 
     def __init__(
             self,
-            data_set: CutSet,
+            cuts: CutSet,
             root_dir: Optional[Pathlike] = None
     ):
         super().__init__()
-        self.data_set = data_set
+        self.cuts = cuts.cuts
         self.root_dir = Path(root_dir) if root_dir else None
-        self.cut_ids = list(self.data_set.cuts.keys())
+        self.cut_ids = list(self.cuts.keys())
 
     def __getitem__(self, idx: int) -> Dict[str, torch.Tensor]:
         cut_id = self.cut_ids[idx]
-        cut = self.data_set.cuts[cut_id]
+        cut = self.cuts[cut_id]
 
         feature = torch.from_numpy(cut.load_features(root_dir=self.root_dir))
 
         return {
             'feature': feature,
-            'text': cut.supervisions.text
+            'text': cut.supervisions[0].text
```

Comment:
what if there's more than one supervision? can you leave a TODO comment here saying that we'll have to work this out for multi-supervision cuts?

Reply:
I think we can assume that there should be only one supervision for each cut in ASR tasks. I put a warning when more supervisions detected.

Reply:
For LibriSpeech, there will be single supervision; however, that won't be true for all ASR tasks. Later on, for corpora like conversations (e.g., SWBD), we'll want to create cuts that might span more than a single utterance, and that's where multiple supervisions will come from. But we can handle it once the need arises, it's fine for now.